src/components/ec-modal/ec-modal.spec.js

```diff
@@ -0,0 +1,163 @@
+/* eslint-disable no-use-before-define */
+import { mount } from '@vue/test-utils';
+import EcModal from './ec-modal.vue';
+
+describe('EcModal', () => {
+  it('should not render the modal', () => {
```

Comment:
I'd write `should not render the modal if <when>`. Because this sounds like EcModal should never render any modal.